bigquery_etl/format_sql/tokenizer.py

```diff
@@ -736,6 +736,33 @@ class Literal(Token):
     )
 
 
+class JinjaExpression(Token):
+    """Jinja expression delimiters {{ }}.
+
+    May be followed by no whitespace or a new line and increased indent.
+    """
+
+    pattern = re.compile(r"{\s*{.*}\s*}")
```

Comment:
I don't think allowing whitespace between the pairs of curly brackets makes sense.  Did you perhaps mean `{{\s*.*\s*}}` to allow newlines before/after the main content?

Also, even with `{{\s*.*\s*}}` that won't allow multiple lines of content because `.` doesn't match newlines by default.

Perhaps something like this instead?
```suggestion
    pattern = re.compile(r"{{.*?}}", re.DOTALL)
```

(these same concerns also apply to `JinjaStatement` and `JinjaComment`)